requirements.yml

```diff
@@ -1,8 +1,8 @@
 ---
-- src: https://github.com/redhat-nfvpe/ansible-role-install-go
+- src: https://github.com/gantsign/ansible-role-golang
   name: install-go
   version: master
-- src: https://github.com/redhat-nfvpe/ansible-role-install-docker
+- src: https://github.com/pmossakx/ansible-role-docker
```

Comment:
Mind leaving this one as our current role? We manage this role for some other projects, too. Thanks

Reply:
Well, I needed to replace it with [my fork supporting proxy for docker](https://github.com/pmossakx/ansible-role-docker/commit/19809fe2448ee0bd51fd0a0a9b4a31cf68765818), maybe I could add it to your role as well?

Reply:
@dougbtv can I add [docker proxy support](https://github.com/pmossakx/ansible-role-docker/commit/19809fe2448ee0bd51fd0a0a9b4a31cf68765818) to https://github.com/redhat-nfvpe/ansible-role-install-docker? This way I won't need to change it to https://github.com/pmossakx/ansible-role-docker